### PR TITLE
Extract button group component from eevee

### PR DESCRIPTION
**Pull request by callumacrae**
# Description

Exactly the same behaviour as in Eevee. The intention is that the collectable button component will use this - there won't be a new model in Contentful (for now, at least).

# Checklist

Pull request contains:

- [x] A new component
- [ ] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [ ] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [ ] PR description includes description of change
- [ ] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.
- [ ] If component change, version updated

### CODE REVIEW

File: src/elements/button-group/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@uswitch/trustyle.button-group",
+  "version": "0.0.0",
```
Comment (Segger87) [inline src/elements/button-group/package.json:3]: is version 0.0.0 ok to have?

Comment (callumacrae) [reply src/elements/button-group/package.json:3]: will be updated by `npx lerna version` before merging :)

**Comment by callumacrae** [comment]
@barneyfoxuk I wasn't sure about that - it's not a collection of elements in itself, just a component that can be used to group other elements.

Happy to change it if you think it's a compound!

**Comment by callumacrae** [comment]
Closing after talking with barney - this is going to stay in Eevee for now.

**Pull request closed without merging**